cvat/apps/dataset_manager/tests/test_rest_api_formats.py

```diff
@@ -365,6 +367,7 @@ def setUpClass(cls):
         super().setUpClass()
         cls.assets_path = osp.join(osp.dirname(__file__), 'assets')
         cls.dumped_files_names = []
+        cls.path = "/tmp/cvat_".join(random.choice(string.ascii_lowercase) for i in range(10))
```

Comment:
Use `datumaro.test_utils.TempDir()` or `tempfile.mkdtemp()` for this. The name this code chooses is not guaranteed to be free.

Reply:
Done. Used from datumaro.util.test_utils.TestDir without determined path.